2018/CG-12-11.md

```diff
@@ -28,6 +28,9 @@ Installation is required, see the calendar invite.
 1. Adoption of the agenda
 1. Proposals and discussions
     1. Review of action items from prior meeting.
+      1. [Follow-up on utf-8 handling in JS spec](https://github.com/WebAssembly/spec/issues/915) (Sven Sauleau)
+      1. The Future of ChakraCore (Limin Zhu)
```

Comment:
Can you link to https://github.com/Microsoft/ChakraCore/issues/5865 as well s.t. people have some context?